src/modules/Authentication/pages/SignUp.js

```diff
@@ -1,19 +1,32 @@
-import React from 'react'
+import React, { useState } from 'react'
 
 import AuthPageFormat from '../components/AuthPageFormat'
 import SignUpForm from '../components/SignUpForm'
 
 export default (props) => {
+  const [showErrorMessage, setShowErrorMessage] = useState(false)
+
   return (
     <AuthPageFormat
       title='Member registration'
       link={{ to: '/auth/login', text: 'Do you already have an account?' }}
+      showErrorMessage={showErrorMessage}
+      onHideErrorMessage={() => setShowErrorMessage(false)}
     >
-      <SignUpForm onSubmit={handleFormSubmit} />
+      <SignUpForm
+        onSuccess={handleSuccess}
+        onError={handleError}
+      />
     </AuthPageFormat>
   )
 
-  function handleFormSubmit (fields) {
-    console.log(fields)
+  function handleSuccess (user) {
+    console.log(user)
```

Comment:
Remove these logs, please.

Reply:
I added the log because the [Block 4](https://github.com/WeDevelop-ARG/full-stack-beginner-program/blob/master/content/block-4/index.md) asked for it.

I'll remove it anyway.